swaybg/main.c

```diff
@@ -74,8 +75,8 @@ static void render_image(struct swaybg_state *state) {
 	cairo_surface_t *image = state->context.image;
 	double width = cairo_image_surface_get_width(image);
 	double height = cairo_image_surface_get_height(image);
-	int wwidth = state->width;
```

Comment:
While you're here can you rename these to `surface_{width,height}`

Reply:
`surface_{width,height}` are unscaled in Wayland terms, so I picked `buffer_{width,height}`